src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java

```diff
@@ -272,24 +290,35 @@ public void should_delete_blob_after_doc_upload_and_mark_envelope_status_as_proc
         given(documentManagementService.uploadDocuments(ImmutableList.of(pdf)))
             .willReturn(getFileUploadResponse());
 
+        given(tokenValidator.getServiceName("testServiceAuthHeader")).willReturn("test_service");
+
         blobProcessorTask.processBlobs();
 
         //Check blob is deleted
         CloudBlockBlob blob = testContainer.getBlockBlobReference(zipFile);
         await().atMost(2, SECONDS).until(blob::exists, is(false));
 
+        //Verify envelope status is updated to DOC_CONSUMED
+        mockMvc.perform(get("/envelopes")
+            .header("ServiceAuthorization", "testServiceAuthHeader"))
+            .andDo(print())
+            .andExpect(status().isOk())
+            .andExpect(content().json(expectedProcessedEnvelopes()));
+
         // Verify envelope status is updated to DOC_PROCESSED
         assertThat(envelopeRepository.findAll())
             .hasSize(1)
             .extracting("status")
-            .containsOnly(DOC_PROCESSED);
+            .containsOnly(DOC_CONSUMED);
```

Comment:
Revert to `DOC_PROCESSED` and check it has been updated to `DOC_CONSUMED` in controller test where this piece belongs to. The processor scheduled task is completely unaware when documents are consumed. Let's keep it that way